src/dns/coredns.rs

```diff
@@ -217,7 +217,8 @@ impl CoreDns {
                                 reply(sender, src_address, &req);
                             } else {
                                 debug!("Not found, forwarding dns request for {:?}", name);
-                                if no_proxy {
+                                let request_name = name.as_str().to_owned();
+                                if no_proxy || request_name.ends_with("dns.podman") || request_name.ends_with("dns.podman.") || request_name.matches('.').count() == 1  {
```

Comment:
I thought the dns search domain is configurable so we should use the same value here.

Reply:
@Luap99 Yes actually I was just testing if CI fails so forgot editing it. Fixed it now.